### PR TITLE
php-fpm: elasticdump for importing/exporting Elasticsearch data

### PR DESCRIPTION
**Pull request by Jancis**
Adds  elasticdump binary for importing/exporting Elasticsearch data

### CODE REVIEW

File: silta-php-fpm/8.2-fpm/Dockerfile [hunk 1]
```diff
@@ -119,6 +119,9 @@ COPY config/w-php-defaults.ini /usr/local/etc/php/conf.d/w-php-defaults.ini
 COPY silta /silta
 COPY silta/entrypoints/10-env.sh /etc/profile.d/10-env.sh
 
+# Elasticdump for importing/exporting Elasticsearch data
+RUN apk add npm nodejs && npm install -g elasticdump
```
Comment (Rade333) [inline silta-php-fpm/8.2-fpm/Dockerfile:123]: Please move this line a bit further up, above lines 95-96 which do cleanup. Also, add `&& npm cache clean --force` after the npm install command.

**Review by Rade333 — CHANGES_REQUESTED** [review]
Looks pretty straight-forward! Added a comment to make image size smaller.

File: silta-php-fpm/8.2-fpm/Dockerfile (hunk 1 above)
Comment (Jancis) [reply silta-php-fpm/8.2-fpm/Dockerfile:123]: was under impression npm is still required. 
code adjusted, works
```bash
$ elasticdump --version
6.111.0
```

Comment (Rade333) [reply silta-php-fpm/8.2-fpm/Dockerfile:123]: Hmm @Jancis I don't see any changes in the code

**Review by Rade333 — CHANGES_REQUESTED** [review]
See https://github.com/wunderio/silta-images/pull/180#discussion_r1701652404

File: silta-php-fpm/8.2-fpm/Dockerfile (hunk 1 above)
Comment (Jancis) [reply silta-php-fpm/8.2-fpm/Dockerfile]: i don't know where they went, recreated :)

**Review by Rade333 — APPROVED** [review]

**Pull request merged**